shared/src/packages/surveys/libs/validation-schemas/survey-request.validation-schema.ts

```diff
@@ -0,0 +1,15 @@
+import joi from 'joi';
```

Comment:
rename file to create-survey.validation-schema.ts, filename should be the same as exported module